pkg/endpoint/restore.go

```diff
@@ -163,7 +163,7 @@ func (e *Endpoint) RegenerateAfterRestore() error {
 	}
 	if buildSuccess := <-e.Regenerate(regenerationMetadata); !buildSuccess {
 		scopedLog.Warn("Failed while regenerating endpoint")
-		return fmt.Errorf("failed while regenerating endpoint")
+		return fmt.Errorf("Failed while regenerating endpoint")
```

Comment:
This does not seem related with the functionality introduced by this PR. It would be nice if code refactoring would be done in a different PR.

Reply:
reverted